File: app/src/main/java/com/dream/bjst/account/ui/AboutUsActivity.java
package com.dream.bjst.account.ui;

import android.view.View;

import com.dream.bjst.R;
import com.dream.bjst.base.BaseActivity;
import com.hjq.bar.TitleBar;

public class AboutUsActivity extends BaseActivity {
     TitleBar mAboutUsTitle;

    @Override
    protected int initLayout() {
        return R.layout.activity_about_us;
    }

    @Override
    protected void initView() {
    mAboutUsTitle=fvbi(R.id.about_us_title);
    }

    @Override
    protected void initData() {
      event();
    }

    private void event() {
        mAboutUsTitle.getLeftView().setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                onBackPressed();
            }
        });
    }
}

File: app/src/main/java/com/dream/bjst/account/ui/AboutUsActivity.kt
package com.dream.bjst.account.ui

import android.os.Bundle
import android.view.View
import com.dream.bjst.account.vm.AboutUsViewModel
import com.dream.bjst.databinding.ActivityAboutUsBinding
import com.tcl.base.common.ui.BaseActivity
import com.tcl.base.kt.ktClick

class AboutUsActivity : BaseActivity<AboutUsViewModel, ActivityAboutUsBinding>() {
    override fun initView(savedInstanceState: Bundle?) {
        TODO("Not yet implemented")
    }

    override fun initData() {
        event()
    }

    private fun event() {
        mBinding.aboutUsTitle.leftView.ktClick() {
            onBackPressed()
        }
    }

    override fun initDataOnResume() {
        TODO("Not yet implemented")
    }
}